src/components/Header.tsx

```diff
@@ -0,0 +1,79 @@
+import { Link } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+import styled from 'styled-components';
+import { tokenState } from '../recoil/atoms/user';
+import { COLOR } from '../utils/color';
+import Icon from './Base/Icon';
+import LinkButton from './LinkButton';
+
+const Header = () => {
+  const token = useRecoilValue(tokenState);
+
+  return (
+    <Container>
+      <Wrapper>
+        <LogoButton to="/">
+          <Logo />
+        </LogoButton>
+        <Icon name="search" size={24} />
+      </Wrapper>
+      {token ? (
+        <Wrapper>
+          <LinkButton to="/" name="new" size={24} />
+          <LinkButton to="/notifications" name="notification" size={24} />
+          <LinkButton to="/profile/me" name="profile" size={24} />
+        </Wrapper>
+      ) : (
+        <Wrapper>
+          <LogInButton to="/login">LOG IN</LogInButton>
+        </Wrapper>
+      )}
+    </Container>
+  );
+};
+
+const Container = styled.div`
+  width: 90%;
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  height: 6rem;
+  align-items: center;
+  position: sticky;
+  top: 0px;
```

Comment:
```suggestion
  top: 0;
```
이렇게 쓰는게 더 좋을 것 같다는 개취를 ... 한 번 남겨봅니다 ㅎㅎ